src/shims/Directory.Build.props

```diff
@@ -19,7 +19,15 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <!-- shared folder for ApiCompat.proj and shims.proj -->
-    <GenFacadesOutputPath>$(ArtifactsObjDir)shims/$(TargetGroup)/</GenFacadesOutputPath>
+    <IsPartialFacadeAssembly>true</IsPartialFacadeAssembly>
+    <GenFacadesIgnoreMissingTypes>true</GenFacadesIgnoreMissingTypes>
+    <IncludeDefaultReferences>false</IncludeDefaultReferences>
+    <HasMatchingContract>true</HasMatchingContract>
   </PropertyGroup>
+
+  <ItemGroup Condition="'$(MSBuildProjectName)' == 'System.Runtime' OR '$(MSBuildProjectName)' == 'mscorlib' OR '$(MSBuildProjectName)' == 'netstandard'">
```

Comment:
Might be worth adding a brief comment here describing why this is necessary

Reply:
I thought I did below and in the file itself?

Reply:
The below comment may seem esoteric to someone unfamiliar with the space, but a longer comment may become a bit much. I'll leave it up to you to decide what's appropriate here.